spec/tasks/changelog_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+return unless RUBY_VERSION >= '2.6'
+
+require_relative '../../tasks/changelog'
+
+# rubocop:disable RSpec/ExampleLength
+RSpec.describe Changelog do
+  subject(:changelog) do
+    list = entries.to_h { |e| [e.path, e.content] }
+    described_class.new(content: <<~CHANGELOG, entries: list)
```

Comment:
I don't see tests making sure that the author links were added to the bottom of the changelog.

Reply:
I didn't implement that because I was lazy, but hey, why not... Done.